src/odemis/driver/test/smaract_test.py

```diff
@@ -720,6 +720,184 @@ def test_parse_tsv_config(self):
         os.remove(file_name)
 
 
+CONFIG_1DOF = {"name": "1DOF",
+        "role": "stage",
+        "ref_on_init": True,
+        "locator": "network:sn:MCS2-00006678",
+        # "locator": "fake",
+        "speed": 0.003,
+        "accel": 0.003,
+        #"hold_time": 1.0,
+        #"pos_deactive_after_ref": True,
+        #"param_file": None,
+        "axes": {
+            'z': {
+                'range': [-20e-3, 20e-3],
+                'unit': 'm',
+                'channel': 0,
+            },
+        },
+}
+
+if TEST_NOHW:
+    CONFIG_1DOF['locator'] = 'fake'
+
+
+class TestMCS2_1DOF(unittest.TestCase):
+    """
+    Tests cases for the SmarAct MCS2 controller
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        cls.dev = smaract.MCS2(**CONFIG_1DOF)
+
+        while not cls.dev.referenced.value:
+            time.sleep(0.1)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.dev.terminate()
+
+    def test_simple(self):
+        self.assertEqual(set(self.dev.axes.keys()), {"z"})
+
+    def test_out_of_range(self):
+        """
+        Test sending a position that is out of range.
+        """
+        pos = {'z': -21e-3}
+        with self.assertRaises(ValueError):
+            self.dev.moveAbs(pos).result()
+
+    def test_move_abs(self):
+        pos1 = {'z': -13.0e-3}
+        pos2 = {'z': -13.01e-3}  # move by -10 um
+        pos3 = {'z': -13.02e-3}  # move by -20 um
+        pos4 = {'z': -12.09e-3}  # move by 10 um
+        pos5 = {'z': -12.08e-3}  # move by 20 um
```

Comment:
I changed a comments a little bit, I will use variable instead of hard coded values. In this case it will give the reader an idea how much each position varies from the pos1 which is a safe position. It can come handy during the hardware tests.